Contest_API.md

```diff
@@ -1226,6 +1226,31 @@ Returned data:
 {"id":"nicky","type":"admin"}
 ```
 
+### Account
+
+Provides information on which account the client represents within a given contest.
+
+The following endpoint is associated with an account:
+
+| Endpoint                 | Mime-type        | Required? | Description
+| :----------------------- | :--------------- | :-------- | :----------
+| `/contests/<id>/account` | application/json | yes       | JSON object representing a single account with properties as defined in the [Accounts](#accounts) table above.
```

Comment:
I would suggest merging the "Account" section into the "Accounts" section. So, there would be three endpoints listed there.

Reply:
Fixed.